src/elements/tip-well/TipWell.spec.js

```diff
@@ -35,12 +35,15 @@ describe('Component: TipWell', () => {
     beforeEach(inject([TipWell], _comp => {
         comp = _comp;
         localStorage.clear();
+        comp.ngOnInit();
```

Comment:
Should not call `ngOnInit()` here. Write a `describe` block for `ngOnInit` and put your checks there. There you can call `ngOnInit`. That way all of the other tests stay pure to their functionality.